### ISSUE TITLE
Loop and Control Flow Challenges:

### ISSUE THREAD

**Issue by yeziarrietty** [yeziarrietty/wenti#3]
What is the difference betweenbreakandcontinuestatements in a loop?

**Comment by Ustinian-yu-code** [comment]
When a break statement is executed in a loop (for or while), it immediately terminates the loop it is currently in. The execution process of the program will jump out of the loop and continue to execute the statements that follow the loop.

When the continue statement is executed in a loop, it skips the remaining blocks of code in the current loop iteration and starts the next iteration of the loop. That is, the loop will not be terminated, but the subsequent code in the current round of the loop will not be executed.
In short, the break statement will terminate the loop early, and the continue statement will stop the loop and move on to the next loop.

**Comment by Evan-Lu-clf** [comment]
1. breakStatement:
      • Thebreakstatement is used to exit the loop immediately, regardless of the loop's original termination condition.
      • When abreakstatement is executed, the control flow jumps to the statement that follows the loop.
      • It is often used when a certain condition is met, and there is no need to continue executing the remaining iterations of the loop.
Example:
【python】
 for i in range(10):
       if i == 5:
           break
       print(i)
   # Output: 0, 1, 2, 3, 4
   # The loop stops when i is 5, and the break statement is executed.
  2. continueStatement:
      • Thecontinuestatement is used to skip the current iteration of the loop and proceed to the next iteration.
      • When acontinuestatement is executed, the remaining code inside the loop for the current iteration is not executed, and the loop immediately checks its condition for the next iteration.
      • It is often used when a certain condition is met, and you want to skip the rest of the code in the loop for that particular iteration but continue with the loop's subsequent iterations.
Example:
【python】
 for i in range(10):
       if i == 5:
           continue
       print(i)
   # Output: 0, 1, 2, 3, 4, 6, 7, 8, 9
   # When i is 5, the continue statement is executed, skipping the print statement for that iteration.
